examples/components-and-props/input-type-file.js

```diff
@@ -0,0 +1,26 @@
+class FileInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit (event) {
```

Comment:
Assuming you are interested in going the `embed` route I mentioned above, you can add line-highlighting into the file by adding a comment above `handleSubmit` like so:
>  // highlight-range{1-4}